docs/source/guide/get_started/introduction.rst

```diff
@@ -7,14 +7,18 @@
 Introduction
 ============
 
-**OpenVINO™ Training Extensions** is a low-code transfer learning framework for Computer Vision. 
+**OpenVINO™ Training Extensions** is a low-code transfer learning framework for Computer Vision.
 
-OpenVINO™ Training Extensions lets users train, infer, optimize and deploy models simply and fast even with low expertise in the deep learning field. OpenVINO™ Training Extensions offers diverse combinations of model architectures, learning methods, and task types based on `PyTorch <https://pytorch.org/>`_ and `OpenVINO™ toolkit <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html>`_. 
+Using the simple CLI commands of the framework, users can  train, infer, optimize and deploy models simply and fast even with low expertise in the deep learning field. OpenVINO™ Training Extensions offers diverse combinations of model architectures, learning methods, and task types based on `PyTorch <https://pytorch.org/>`_ and `OpenVINO™ toolkit <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html>`_.
```

Comment:
```suggestion
Using the simple CLI commands of the framework, users can train, infer, optimize and deploy models simply and fast even with low expertise in the deep learning field. OpenVINO™ Training Extensions offers diverse combinations of model architectures, learning methods, and task types based on `PyTorch <https://pytorch.org/>`_ and `OpenVINO™ toolkit <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html>`_.
```
double space -> single space

Reply:
What do you think of Using the simple CLI commands of the framework  ->  With the simple CLI commands of the framework